build.sbt

```diff
@@ -310,15 +310,12 @@ def addCommandsAlias(name: String, values: List[String]) =
 addCommandsAlias(
   "validate",
   List(
```

Comment:
Sadly, these changes mean you can no longer just run `sbt validate` (you have to run `sbt +validate`) locally to check everything passes.